src/c/test/smoketest.cpp

```diff
@@ -0,0 +1,50 @@
+/************************************************************\
+ * Copyright 2021 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#include <cstdio>
+#include <string>
+#include <unistd.h>
+
+__attribute__((annotate("@critical_path(pointcut=\"after\", scope=\"bar\")")))
```

Comment:
Thoughts on also supporting single-quotes within the annotation?

```suggestion
__attribute__((annotate("@critical_path(pointcut='after', scope='bar')")))
```

The escaped double-quotes aren't a big deal, but I think the ability to also use an unescaped single-quote would be nice.  I assume it would just require tweaking the regex in the yak/bison for the label to something like:

```diff
- label              ["]([^"\\\n]|\\(.|\n))*["]
+ label              ["']([^"'\\\n]|\\(.|\n))*["']
```

Reply:
Oh.... good idea!

Reply:
I realize this will validate quote mismatches as well. (pointcut='after\") is valid. You can avoid this but it feels like we may just stick with single quotes as the only valid gramma...